front-project/src/routes/SignUpPage.jsx

```diff
@@ -1,5 +1,10 @@
 import SignUpForm from "../components/SignUpForm";
+import { Navigate } from "react-router-dom";
+import { useSelector } from "react-redux";
 
 export const SignUpPage = () => {
-  return <SignUpForm />;
+  const user = useSelector((state) => state.auth.user);
```

Comment:
Write all selectors in separate files, so these function will be reusable and it'll be easier to refactor and change logic (for example if we decide to change field "user" to "currentUser"). 

In this case:
create file
store/auth/auth.selectors.js

add selector function
const getUser = state => state.auth.user
const getIsLoggedIn = state => Boolean(getUser(state))